supervisor/updater.py

```diff
@@ -180,24 +186,37 @@ async def fetch_data(self):
         url = URL_HASSIO_VERSION.format(channel=self.channel)
         machine = self.sys_machine or "default"
 
+        # Get data
         try:
             _LOGGER.info("Fetching update data from %s", url)
             async with self.sys_websession.get(url, timeout=10) as request:
-                data = await request.json(content_type=None)
+                data = await request.read()
 
         except (aiohttp.ClientError, asyncio.TimeoutError) as err:
             self.sys_supervisor.connectivity = False
-            _LOGGER.warning("Can't fetch versions from %s: %s", url, err)
-            raise UpdaterError() from err
+            raise UpdaterError(
```

Comment:
We should split this.
Timeout error does not have anything for `err`.

Reply:
will respect that on next PR which improve that all anyway